src/components/WYSIWYGInput/index.tsx

```diff
@@ -227,8 +227,6 @@ export const WYSIWYGInput = forwardRef<WYSIWYGInputRef, WYSIWYGInputProps>(
       };
 
       if (!isEditorStateInitialized) {
-        console.log('resetting');
-        // setEditorState(EditorState.set(editorState, { decorator: defaultDecorators }));
```

Comment:
Is this expected or was it just for testing?

Reply:
it was just for testing on the last version that delivered, only noticed after I made the last release, and I forgot to remove so I did it now :)